Doc/library/pathlib.rst

```diff
@@ -1592,6 +1592,18 @@ Copying, renaming and deleting
    .. versionadded:: 3.14
 
 
+.. method:: Path.move(target)
+
+   Recursively move this file or directory tree to the given *target*, and
+   return a new :class:`!Path` instance pointing to *target*.
+
+   If both paths are on the same filesystem, the move is performed with
+   :func:`os.replace`. Otherwise, this path is copied (preserving metadata and
```

Comment:
What happens if target is the same as the source file? does it raise like for `mv` or is a no-op?

Reply:
It raises - I've added a mention in the paragraph above.